### PR TITLE
build: use proper operator-sdk on ARM64

### PR DESCRIPTION
**Pull request by synarete**
Align operator-sdk download URL with instructions from https://sdk.operatorframework.io/docs/installation/

Build tested on AWS/Graviton machine (arm64).

Signed-off-by: Shachar Sharon <ssharon@redhat.com>

### Explain the changes
1. Use proper ARCH tag when using operator-sdk (do not asume hard-coded x86_64)

### Issues: Fixed #xxx / Gap #xxx
1. Build noobaa-operator image on arm64 machine (AWS Graviton)

### Testing Instructions:
1. AWS aarch64 vm

- [ ] Doc added/updated
- [ ] Tests added

### CODE REVIEW

**Comment by liranmauda** [comment]
@dannyzaken We need to see if this works on M1 Mac...

**Comment by dannyzaken** [comment]
@synarete on m1 mac `uname -m` returns `arm64`. also, there is no v0.17.2 arm build for mac.
Can you please modify the script to continue using x86 when on Darwin? M1 macs have a translation layer, so it will continue to work.

**Comment by synarete** [comment]
> 

@dannyzaken Fair enough, thanks for this input. I will fix the script accordingly.

**Comment by synarete** [comment]
> @synarete on m1 mac `uname -m` returns `arm64`. also, there is no v0.17.2 arm build for mac. Can you please modify the script to continue using x86 when on Darwin? M1 macs have a translation layer, so it will continue to work.

@dannyzaken Please check with the newly modified version (I do not have access to Apple's M! mac hw)

**Comment by liranmauda** [comment]
Just a comment, Upstream builds and downstream builds are often not the same.
We need to verify also that the downstream builds Support ARM64 
adding @b-ranto

**Comment by b-ranto** [comment]
@liranmauda The support for aarch64 (ARM64) is planned for ODF 4.13, we have the architecture enabled for the product but we don't really do aarch64 builds, yet.

**Comment by nimrod-becker** [comment]
@b-ranto @synarete 
Please note that at least for d/s aarch64 (whenever we will build) we also need to build an appropriate RPM (CLI)

**Comment by synarete** [comment]
> @b-ranto @synarete Please note that at least for d/s aarch64 (whenever we will build) we also need to build an appropriate RPM (CLI)

@nimrod-becker Indeed so. And there few more challenges on noobaa-core as well (requires a separate PR).

**Comment by dannyzaken** [comment]
@synarete works on my M1 Mac

**Review by liranmauda — COMMENTED** [review]
LGTM, once it is rebased we can approve.

**Review by liranmauda — APPROVED** [review]
Once it will pass tests we can merge it.

**Pull request merged**
